src/integrations/third-party/wincher.php

```diff
@@ -46,34 +70,64 @@ public function add_integration_toggle( $integration_toggles ) {
 		if ( \is_array( $integration_toggles ) ) {
 			$integration_toggles[] = (object) [
 				/* translators: %s: 'Wincher' */
-				'name'    => \sprintf( \__( '%s integration', 'wordpress-seo' ), 'Wincher' ),
-				'setting' => 'wincher_integration_active',
-				'label'   => \sprintf(
+				'name'     => \sprintf( \__( '%s integration', 'wordpress-seo' ), 'Wincher' ),
+				'setting'  => 'wincher_integration_active',
+				'label'    => \sprintf(
 				/* translators: %s: 'Wincher' */
 					\__( 'The %s integration offers the option to track specific keyphrases and gain insights in their positions.', 'wordpress-seo' ),
 					'Wincher'
 				),
-				'order'   => 11,
+				'order'    => 11,
+				'disabled' => ( $this->wincher->is_disabled() ),
 			];
 		}
 
 		return $integration_toggles;
 	}
 
 	/**
-	 * Loads additional content for the passed integration.
-	 *
-	 * @param Object $integration The integration object.
+	 * Adds the disabled note when the integration toggle is disabled.
 	 *
-	 * @return void
+	 * @param Yoast_Feature_Toggle $integration The integration toggle class.
 	 */
-	public function load_toggle_additional_content( $integration ) {
-		switch ( $integration->setting ) {
-			case 'wincher_integration_active':
+	public function after_integration_toggle( $integration ) {
+		if ( $integration->setting === 'wincher_integration_active' ) {
+
+			// Check if 'WPSEO_PATH' is defined else the relevant unittest will require the file.
+			if ( defined( 'WPSEO_PATH' ) ) {
 				require \WPSEO_PATH . 'admin/views/tabs/metas/paper-content/integrations/wincher.php';
-				break;
-			default:
-				break;
+			}
+
+			if ( $integration->disabled ) {
+
+				$conditional = $this->wincher->is_disabled( true );
+
+				if ( $conditional === 'Non_Multisite_Conditional' ) {
+					$this->get_disabled_note();
+				}
+			}
```

Comment:
Do you care about all this?
I think it could be replaced with:
```suggestion
			if ( \is_multisite() ) {
				$this->get_disabled_note();
			}
```
Though granted this makes it too "smart" to know about the multisite bit. It is very much simpler, and takes away the need for that `is_disabled` actually.